examples/ga_peoplecount/package.json

```diff
@@ -12,6 +12,6 @@
   "dependencies": {
```

Comment:
I think this example is missing a ```await detector.init();```

Reply:
Where, which file?

Reply:
Here https://github.com/Huddly/sdk/blob/master/examples/ga_peoplecount/ga_peoplecount.js#L21